src/input.cpp

```diff
@@ -647,133 +655,46 @@ void Input::check_lammps_command( int line )
-                    {
-                        n++;
-                    }
-                }
-            }
-        }
-        system->N_local = n;
-        system->N = n;
-        system->resize( n );
-        s = *system;
-        auto x = Cabana::slice<Positions>( s.xvf );
-        auto v = Cabana::slice<Velocities>( s.xvf );
-        auto id = Cabana::slice<IDs>( s.xvf );
-        auto type = Cabana::slice<Types>( s.xvf );
-        auto q = Cabana::slice<Charges>( s.xvf );
 
-        n = 0;
-        for ( T_INT iz = iz_start; iz <= iz_end; iz++ )
-        {
-            T_FLOAT ztmp = lattice_constant * ( iz + lattice_offset_z );
-            for ( T_INT iy = iy_start; iy <= iy_end; iy++ )
-            {
-                T_FLOAT ytmp = lattice_constant * ( iy + lattice_offset_y );
-                for ( T_INT ix = ix_start; ix <= ix_end; ix++ )
-                {
-                    T_FLOAT xtmp = lattice_constant * ( ix + lattice_offset_x );
-                    if ( ( xtmp >= s.sub_domain_lo_x ) &&
-                         ( ytmp >= s.sub_domain_lo_y ) &&
-                         ( ztmp >= s.sub_domain_lo_z ) &&
-                         ( xtmp < s.sub_domain_hi_x ) &&
-                         ( ytmp < s.sub_domain_hi_y ) &&
-                         ( ztmp < s.sub_domain_hi_z ) )
-                    {
-                        x( n, 0 ) = xtmp;
-                        x( n, 1 ) = ytmp;
-                        x( n, 2 ) = ztmp;
-                        type( n ) = rand() % s.ntypes;
-                        id( n ) = n + 1;
-                        n++;
-                    }
-                }
-            }
-        }
-        comm->reduce_int( &system->N, 1 );
-
-        // Make ids unique over all processes
-        T_INT N_local_offset = n;
-        comm->scan_int( &N_local_offset, 1 );
-        for ( T_INT i = 0; i < n; i++ )
-            id( i ) += N_local_offset - n;
-
-        if ( system->do_print )
-            printf( "Atoms: %i %i\n", system->N, system->N_local );
-    }
-
-    // Create Face Centered Cubic (FCC) Lattice
-    if ( lattice_style == LATTICE_FCC )
-    {
-        system->domain_x = lattice_constant * lattice_nx;
-        system->domain_y = lattice_constant * lattice_ny;
-        system->domain_z = lattice_constant * lattice_nz;
-        system->domain_hi_x = system->domain_x;
-        system->domain_hi_y = system->domain_y;
-        system->domain_hi_z = system->domain_z;
-
-        comm->create_domain_decomposition();
-        s = *system;
-
-        double basis[4][3];
-        basis[0][0] = 0.0;
-        basis[0][1] = 0.0;
-        basis[0][2] = 0.0;
-        basis[1][0] = 0.5;
-        basis[1][1] = 0.5;
-        basis[1][2] = 0.0;
-        basis[2][0] = 0.5;
-        basis[2][1] = 0.0;
-        basis[2][2] = 0.5;
-        basis[3][0] = 0.0;
-        basis[3][1] = 0.5;
-        basis[3][2] = 0.5;
-
-        for ( int i = 0; i < 4; i++ )
+        double basis[4][3] = {
```

Comment:
This is much better!

Reply:
I can't believe that we're relying on a machine to tell us where to put whitespace in C++ code... They don't have eyes, and have never used tab or shift keys.